backend/src/components/applications.js

```diff
@@ -80,9 +106,70 @@ async function createApplication(req, res) {
   }
 }
 
+async function getSupplementaryApplications(req, res) {
+  try {
+    //const operation = `ofm_applications(${req.params.applicationId})`
+    const operation = `ofm_allowances?$filter=(_ofm_application_value eq ${req.params.applicationId} and statuscode eq 1)`
+    const response = await getOperation(operation)
+    log.info('resp')
+    log.info(response)
+    return res.status(HttpStatus.OK).json(mapSupplementaryApplicationObjectForFront(response.value))
+    //return res.status(HttpStatus.OK).json({ test: 'test1' })
+  } catch (e) {
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(e.data ? e.data : e?.status)
+  }
+}
+
+async function createSupplementaryApplication(req, res) {
+  log.info('i got here')
```

Comment:
Probably a debugging artifact? same comments with the commented out code from above.